Sources/ForageSDK/Foundation/Telemetry/ForageLogger.swift

```diff
@@ -78,17 +78,24 @@ internal protocol ForageLogger {
     /// - Parameter error: An optional error object associated with the critical log.
     /// - Parameter attributes: An optional dictionary of key-value pairs providing additional context for the log.
     func critical(_ message: String, error: Error?, attributes: [String: Encodable]?)
+    
+    /// The ForageSDK object needs a handler to the TraceId in order to pass it along to each endpoint call.
+    /// - Returns: The traceID that was generated at the beginning of the app's lifecycle.
+    func getTraceId() -> String
```

Comment:
Nit: should be refactored to `getTraceID` for swift conventions; but feel free to ignore this, since this code is internal